modules/proc_deploy_quay_single.adoc

```diff
@@ -51,7 +51,7 @@ outside of the local system, run the following commands (add `--zone=<yourzone>`
 # firewall-cmd --reload
 ....
 
-. **Install / Deploy a Database**: Choose either MySQL or PostgreSQL as a database. This example shows how to deploy the link:https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/mysql-57-rhel7[MySQL database container] (see the link:https://access.redhat.com/documentation/en-us/red_hat_software_collections/2/html-single/using_red_hat_software_collections_container_images/#mysql[MySQL] section of Using Red Hat Software Collections Container Images for details.) To configure the MySQL database, you can use the values shown here or change any of the following for storing MySQL data (/var/lib/mysql) and setting database values:
+. **Install / Deploy a Database**: Choose either MySQL or PostgreSQL as a database. This example shows how to deploy the link:https://access.redhat.com/containers/#/registry.access.redhat.com/rhel8/mysql-80[MySQL database container]. To configure the MySQL database, you can use the values shown here or change any of the following for storing MySQL data (/var/lib/mysql) and setting database values:
```

Comment:
Should point out that MySQL 8 requires that the database is explicitly created with the latin1 character set (it is the default in 5.x):
mysql> create database quay character set latin1;
Query OK, 1 row affected (0.02 sec)

Reply:
@BillDett  Already mentioned here: https://github.com/quay/quay-docs/pull/102/files#diff-523a76a316b41acdde893295b124efd7d66f918bcd16afae0ad7c6cecbe07f07R89

Reply:
instructions at https://github.com/quay/quay-docs/blob/master/modules/proc_deploy_quay_single.adoc